src/coreclr/jit/emitarm64.cpp

```diff
@@ -30320,6 +30369,26 @@ void emitter::emitDispInsHelp(
             break;
         }
 
+        // <Zd>.<T>, <Zn>.<T>[<imm>]
+        // <Zd>.<T>, <V><n>
+        case IF_SVE_BW_2A: // ........ii.xxxxx ......nnnnnddddd -- SVE broadcast indexed element
+            imm = emitGetInsSC(id);
+            emitDispSveReg(id->idReg1(), id->idInsOpt(), true); // ddddd
+            if (imm > 0)
+            {
+                emitDispSveReg(id->idReg2(), id->idInsOpt(), false); // nnnnn
+                emitDispElementIndex(imm, false);
+            }
+            else if (imm == 0)
+            {
+                emitDispReg(id->idReg2(), optGetSveElemsize(id->idInsOpt()), false);
+            }
+            else
+            {
+                unreached();
```

Comment:
Nit: We don't typically do any validation in `emitDispInsHelp`, but I think this would read better if you converted this if-elseif-else into an if-else, and just asserted that `imm == 0` in the else block. Should the assert hit, we'd see the `imm == 0` check failed in the console, instead of the more generic `unreached()` message.